src/github_refresh.py

```diff
@@ -78,27 +82,46 @@ def get_epv_list():
             if bool(dep_list):
                 for k in dep_list:
                     name = k.split(" ")[0]
-                    epv_list.append({
-                        "ecosystem": eco,
-                        "name": name
-                    })
+                    epv_list[eco].append(name)
             else:
                 logger.info("No deps found in the report for the ecosystem {}".format(eco))
+
+    logger.info("Fetching venus V2 stack report for the date {}".format(yest_date))
+    v2_stack_json = retrieve_dict("v2/daily/" + yest_date + ".json")
+    if v2_stack_json:
+        for eco in eco_list:
+            dep_list = v2_stack_json.get('stacks_summary', {}).get(eco, {}) \
+                .get('unique_dependencies_with_frequency', {})
+            if bool(dep_list):
+                for k in dep_list:
+                    name = k.split(" ")[0]
+                    epv_list[eco].append(name)
+            else:
+                logger.info("No deps found in the report for the ecosystem {}".format(eco))
+    epv_list = {
```

Comment:
Why is this required, is it to remove duplicates?

Reply:
yes

Reply:
Can you not create a set in the first place and try adding to it.